internal/persistence/sql/relationtuples.go

```diff
@@ -210,8 +210,9 @@ func (p *Persister) GetRelationTuples(ctx context.Context, query *relationtuple.
 	}
 
 	sqlQuery := p.QueryWithNetwork(ctx).
-		Order("nid, namespace, object, relation, subject_id, subject_set_namespace, subject_set_object, subject_set_relation, commit_time").
-		Paginate(pagination.Page, pagination.PerPage)
+		Order("shard_id, nid").
```

Comment:
Since `shard_id` is already unique, just ordering by that should suffice, right?

Reply:
Generally yes, but the primary key is `("shard_id, "nid")`. Not all databases use index prefixes, so using this ensures good performance across all of them.

Reply:
Thanks!